engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -870,6 +867,18 @@ private void handleVolumeMigrationForKVM(VolumeInfo srcVolumeInfo, VolumeInfo de
                 throw new CloudRuntimeException(errMsg, ex);
             }
         } finally {
+            // revoke access (for managed volumes)
```

Comment:
Yes, I would expect each storage provider to test the PR or RC for the release with this code.  Generally there were several places where the grant and revoke access was applied inconsistently.  We discovered this with our changes to do per-host VLUN assignments in Pure and Primera and cleaned it up since it became absolutely required any time a volume was added or removed on a host where a VM was being scheduled.